README-developers.md

```diff
@@ -10,3 +10,113 @@
 
 ### productivity
 - set up git [aliases](http://githowto.com/aliases)
+-
+### Branch and Release Management
+
+The latest development code lives in the `master` branch. It should be the
+primary branch for the repository. The primary branch will be the default branch
+checked out when user does `git clone`. Pull requests are merged directly to the
+`master` branch.
+
+#### Public Testing Servers
+
+We are planing to setup GitHub actions to deploy all merges to the testing
+server. Users are invited to test the latest changes on the testing server.
+Volunteers become our quality assurance team. They help us detect bugs before
+changes reach the production servers.
+
+#### Deployment to the Production Servers
+
+The component lead tags the testing image with a production tag when they think
+the testing deployment has had enough testing. The lead then deploys the new
+production image to the production servers.
+
+### Development Work in a Personal Fork Repository
+
+#### Cloning the Primary Repository
+
+You can clone the primary repository normally. This document shows example
+commands for this repository.
+
+`git clone https://github.com/online-bridge-hackathon/common-docs.git`
+
+#### Creating a Personal Fork
+
+Go to the component page in the GitHub website. You can follow the
+[Forking Projects](https://guides.github.com/activities/forking/) guide.
+
+#### Adding the Personal Fork as a remote
+
+Git supports multiple remote repositories for a local working tree. `git remote`
+command can be used to manage remote repositories. In most cases,
+`git remote add` is the only command required.
+
+`git remote add personal https://github.com/<username>/common-docs.git`
```

Comment:
(Did I leave a comment for this already? I thought I had.)

I'd like to see context for this.

1. Am I in a local copy of my forked repo when I run the command? 
2. Note that the remote repo name is just an example.
3. This particular remote repo has been deleted.

Reply:
> I'd like to see context for this.
> 
> 1. Am I in a local copy of my forked repo when I run the command?

Maybe this makes the context clear:
"`git remote` commands require working directory inside a local repository created by `git clone`."

> 2. Note that the remote repo name is just an example.

Example and hint how to check real name:
"URL is an example based on default GitHub naming for a fork. You get the
repository URL for your fork from the GitHub repository page."

> 3. This particular remote repo has been deleted.

Fixed the stale repository to bridge-hackathon.git

Reply:
Better, but which repo should I have cloned to start, the main repo or my fork? Does `git remote add` create a fork or connect to an existing fork? I'm hoping to see a series of steps that a new developer can follow in order to start contributing, ideally referring to documentation only if he hits a stumbling block or wants to understand what's going on 'under the hood'.

Reply:
> Better, but which repo should I have cloned to start, the main repo or my fork? Does `git remote add` create a fork or connect to an existing fork? I'm hoping to see a series of steps that a new developer can follow in order to start contributing, ideally referring to documentation only if he hits a stumbling block or wants to understand what's going on 'under the hood'.

The clone is the previous step defined two sections ago. I was hoping that ordering would make it clear orders is:
1. Clone primary repo
2. Clock the web link to create a fork
3. Add the work as a second remote to the existing local repository.

Reply:
Some typos?

If I've interpreted correctly then this order is clear. I haven't checked the doc in context.

I'll know better how easy the process is to follow when I try it from scratch.